### ISSUE TITLE
Version 4.5.2 offered via ionic doctor check but not in GIT

### ISSUE THREAD

**Issue by corran2go** [ionic-team/ionic-native#2284]
**I'm submitting a ...**  (check one with "x")
[X] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://ionicworldwide.herokuapp.com/

**Current behavior:**
Version 4.5.2 is offered and installed from npm via `ionic doctor check` but only 4.5.1 is available in this GIT Repo.

**Expected behavior:**
GIT and npm versions have to match

**Steps to reproduce:**
run `ionic doctor check` with 4.5.1 installed:

```
[WARN] Update available for Ionic Native.

       An update is available for Ionic Native (4.5.1 => 4.5.2). To fix, the following
       step(s) need to be taken:

            1) Run: npm i --save -E @ionic-native/app-version@4.5.2
            2) Run: npm i --save -E @ionic-native/camera@4.5.2
            3) Run: npm i --save -E @ionic-native/core@4.5.2
            4) Run: npm i --save -E @ionic-native/device@4.5.2
            5) Run: npm i --save -E @ionic-native/email-composer@4.5.2
            6) Run: npm i --save -E @ionic-native/file@4.5.2
            7) Run: npm i --save -E @ionic-native/file-transfer@4.5.2
            8) Run: npm i --save -E @ionic-native/http@4.5.2
            9) Run: npm i --save -E @ionic-native/in-app-browser@4.5.2
            10) Run: npm i --save -E @ionic-native/splash-screen@4.5.2
            11) Run: npm i --save -E @ionic-native/status-bar@4.5.2


? Fix automatically? Yes
> npm i --save -E @ionic-native/app-version@4.5.2
...
```

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
cli packages: (/home/sw/.nvm/versions/node/v8.9.4/lib/node_modules)

    @ionic/cli-utils  : 1.19.1
    ionic (Ionic CLI) : 3.19.1

global packages:

    cordova (Cordova CLI) : 8.0.0

local packages:

    @ionic/app-scripts : 3.1.8
    Cordova Platforms  : none
    Ionic Framework    : ionic-angular 3.9.2

System:

    Android SDK Tools : 26.1.1
    Node              : v8.9.4
    npm               : 5.6.0
    OS                : Linux 4.4

Environment Variables:

    ANDROID_HOME : /home/sw/Android/Sdk

Misc:

    backend : legacy
```

**Comment by corran2go** [comment]
It would be great to get some consistent version information, as this gets quite messy in a big project:

Now https://github.com/ionic-team/ionic-native/releases lists v4.5.1
https://github.com/ionic-team/ionic-native/blob/master/CHANGELOG.md lists 4.5.2
and ionic doctor check / npm lists `An update is available for Ionic Native (4.5.2 => 4.5.3)`

**Comment by etchen99** [comment]
What does the npm package v4.5.3 correspond to in this repo since v4.5.3 isn't available here?
